app/helpers/login_helper.rb

```diff
@@ -0,0 +1,2 @@
+module LoginHelper
+end
```

Comment:
if this helper are empty please remove it

Reply:
all the files?